barretenberg/cpp/src/barretenberg/crypto/merkle_tree/lmdb_store/functions.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "barretenberg/crypto/merkle_tree/types.hpp"
+#include "barretenberg/numeric/uint128/uint128.hpp"
+#include "barretenberg/numeric/uint256/uint256.hpp"
+#include <cstdint>
+#include <lmdb.h>
+#include <vector>
+
+namespace bb::crypto::merkle_tree {
```

Comment:
functions.hpp is a  generic, is this maybe a 'callbacks.hpp'?

Reply:
Done